Changelog: back to school 2024 update (#370)
CleverCloud/documentation: content/changelog/2024-07-30-github-action-1.1.md

---
title: "Version 1.1 of our GitHub Action for review applications is available"
date: 2024-07-30
tags:
  - github
  - action
authors:
  - name: David Legrand
    link: https://github.com/davlgd
    image: https://github.com/davlgd.png?size=40
  - name: Julia March
    link: https://github.com/juliamrch
    image: https://github.com/juliamrch.png?size=40
description: Better, simpler GitHub Action
excludeSearch: true
---

> We updated the [GitHub Action](https://github.com/marketplace/actions/clever-cloud-review-app-on-prs) to deploy applications from GitHub to Clever Cloud. Version 1.1 simplifies workflow files and gives conditional comments depending on the actions performed on the PR (deploy, update, delete).

Reviewer: ```suggestion
Update for [GitHub Action](https://github.com/marketplace/actions/clever-cloud-review-app-on-prs) to deploy applications from GitHub to Clever Cloud. Version 1.1 simplifies workflow files and gives conditional comments depending on the actions performed on the PR (deploy, update, delete).
```
